fuse/dir.go

```diff
@@ -34,7 +34,7 @@ func (d *dir) children(ctx context.Context) (map[string]plugin.Entry, error) {
 
 	// Cache List requests. FUSE often lists the contents then immediately calls find on individual entries.
 	if plugin.ListAction().IsSupportedOn(updatedEntry) {
-		return plugin.CachedList(ctx, updatedEntry.(plugin.Parent))
+		return plugin.List(ctx, updatedEntry.(plugin.Parent))
```

Comment:
It'd be nice if there's an easy way to detect when we're auto-completing vs. invoking something. That way, we don't report analytics for auto-complete (unless we want to).

Reply:
I'm not concerned about reporting for auto-complete.